amlip_docs/rst/demo/tensor_inference.rst

```diff
@@ -0,0 +1,144 @@
+.. include:: ../exports/alias.include
+
+.. _demo_inference:
+
+#########################
+TensorFlow Inference Demo
+#########################
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 2
+
+Background
+==========
+
+This demo shows how to implement 2 types of nodes, :ref:`user_manual_nodes_inference` and :ref:`user_manual_nodes_edge`, to perform TensorFlow inference on a given image.
+
+With these 2 nodes implemented, the user can deploy as many nodes of each kind as desired and check the behaviour of a simulated |amlip| network running.
+
+
+Inference refers to the process of using a trained model to make predictions or draw conclusions based on input data.
+It involves applying the learned knowledge and statistical relationships encoded in the model to new, unseen data.
+
+The inference of an image involves passing the image through a trained AI model to obtain a classification based on the learned knowledge and patterns within the model.
+
+TensorFlow
+----------
+
+`TensorFlow <https://www.tensorflow.org/>`_ is an end-to-end machine learning platform with pre-trained models.
+
+Edge Node
+---------
+
+This node simulates an :ref:`user_manual_nodes_edge`.
```

Comment:
This would make much more sense with a diagram. Saying that EdgeNode is an EdgeNode seems a bit redundant if there is no context (context could be the diagram)